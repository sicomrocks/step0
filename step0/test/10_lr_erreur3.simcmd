#Test de la fonction lr : mauvaise valeur de paramètre

lr $12 0xffffffff
lr $12 0x5661bf656	#le nombre dépasse 8 digits
lr $12 0x123
