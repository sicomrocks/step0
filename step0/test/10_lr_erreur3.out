12 : 0x7fffffff
